.azure-pipelines-templates/test.yml

```diff
@@ -10,6 +10,22 @@ steps:
   displayName: CTest
   workingDirectory: build
 
+# If a test fails, crudely upload nodes' logs for all tests
+- task: CopyFiles@2
+  inputs:
+    contents: '**/?(out|err)'
```

Comment:
This is probably outside the scope of this PR but should we produce core dumps on failures and then copy the core files?

Reply:
For the cores to be useful, we’d need to keep the binaries. Which may be fine but is a non trivial amount of storage.

Reply:
So I originally tried to do this but this results in quite a lot of storage used (~130 MB per run). See https://dev.azure.com/MSRC-CCF/CCF/_build/results?buildId=5172&view=artifacts&type=publishedArtifacts